Source/WebCore/platform/audio/cocoa/AudioSampleBufferConverter.mm

```diff
@@ -235,16 +234,32 @@
             size = sizeof(outputBitRate);
             if (auto error = PAL::AudioConverterSetProperty(m_converter, kAudioConverterEncodeBitRate, size, &outputBitRate))
                 RELEASE_LOG_ERROR(MediaStream, "AudioSampleBufferConverter setting default kAudioConverterEncodeBitRate failed with %d", static_cast<int>(error));
+            else
+                m_defaultBitRate = outputBitRate;
         }
     }
 
+    if (isPCM() && m_options.preSkip) {
+        AudioConverterPrimeInfo primeInfo = { static_cast<UInt32>(*m_options.preSkip), 0 };
+        if (auto error = PAL::AudioConverterSetProperty(converter, kAudioConverterPrimeInfo, sizeof(primeInfo), &primeInfo))
+            RELEASE_LOG_ERROR(MediaStream, "AudioSampleBufferConverter setting default kAudioConverterPrimeInfo failed with %d", static_cast<int>(error));
+    }
+
+    if (!isPCM()) {
+        AudioConverterPrimeInfo primeInfo { 0, 0 };
+        UInt32 size = sizeof(primeInfo);
+        if (auto error = PAL::AudioConverterGetProperty(m_converter, kAudioConverterPrimeInfo, &size, &primeInfo))
+            RELEASE_LOG_ERROR(MediaStream, "AudioSampleBufferConverter getting kAudioConverterPrimeInfo failed with %d", static_cast<int>(error));
+        m_remainingPrimeDuration = PAL::CMTimeMake(primeInfo.leadingFrames, m_destinationFormat.mSampleRate);
```

Comment:
Do we really want to set `m_remainingPrimeDuration` when the call to `AudioConverterGetProperty` failed?

Reply:
> Do we really want to set `m_remainingPrimeDuration` when the call to `AudioConverterGetProperty` failed?

Yes, it will be set to 0.
That's how we can then simply assess that m_remainingPrimeDuration is never invalid